lib/pbench/server/api/resources/query_apis/datasets_list.py

```diff
@@ -120,30 +120,29 @@ def postprocess(self, es_json: Dict[AnyStr, Any]) -> Dict[AnyStr, Any]:
         datasets = []
         hits = es_json["hits"]["hits"]
         self.logger.info("{} controllers found", len(hits))
+        controller = None
         for dataset in hits:
             src = dataset["_source"]
             run = src["run"]
+            if controller:
+                if controller != run["controller"]:
+                    self.logger.warning(
+                        "Expected all controllers to match: {} and {} don't match",
+                        controller,
+                        run["controller"],
+                    )
+            else:
+                controller = run["controller"]
             d = {
-                "key": run["name"],
-                "run.name": run["name"],
-                "run.controller": run["controller"],
-                "run.start": run["start"],
-                "run.end": run["end"],
+                "result": run["name"],
+                "controller": run["controller"],
+                "start": run["start"],
+                "end": run["end"],
                 "id": run["id"],
             }
-            try:
-                timestamp = parser.parse(run["start"]).utcfromtimestamp()
-            except Exception as e:
-                self.logger.info(
-                    "Can't parse start time {} to integer timestamp: {}",
-                    run["start"],
-                    e,
-                )
-                timestamp = dataset["sort"][0]
 
-            d["startUnixTimestamp"] = timestamp
             if "config" in run:
-                d["run.config"] = run["config"]
+                d["config"] = run["config"]
             if "prefix" in run:
                 d["run.prefix"] = run["prefix"]
```

Comment:
You've removed the other instances of the `run.` key prefix; should it be removed from here, too?

Reply:
Nope; this prefix is in the original dashboard code, but I'd missed that `run["config"]` was stored as just `"config"`. I don't even know where I got the timestamp stuff from; I swear I saw it in the dashboard, but when I started double-checking today I couldn't find any trace of such a field. Weird.

I was more uncertain about dropping `"key"` ... it's __not__ in the dashboard, however the dashboard has a lot of React complaints that array elements should have unique `"key"` values, and I might have added that one deliberately in an attempt to quiet them. However I'm not sure that actually helped anything, the dashboard code itself doesn't need it, and I decided to stick to the original dashboard model for this pass.